GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -146,13 +146,15 @@ class ChatView: EngagementView {
         tableView.estimatedRowHeight = UITableView.automaticDimension
         tableView.separatorStyle = .none
         tableView.contentInset = kChatTableViewInsets
+        tableView.keyboardDismissMode = .onDrag
         tableView.register(cell: ChatItemCell.self)
 
         unreadMessageIndicatorView.tapped = { [weak self] in
             self?.scrollToBottom(animated: true)
         }
 
         observeKeyboard()
+        addTapGesture()
```

Comment:
Maybe we could give this function a more meaningful name, one that includes the purpose of that tap gesture? E.g. `addKeyboardDismissalTapGesture()`. It's a mouthful, but it's very descriptive 😄

Reply:
Changed, thanks!